ui/page/root/home_page.go

```diff
@@ -407,6 +409,13 @@ func (hp *HomePage) LayoutTopBar(gtx C) D {
 	padding20 := values.MarginPadding20
 	padding10 := values.MarginPadding10
 
+	topBottomPadding := padding10
+	// Remove top and bottom padding if on mobile view and on the SingleWalletMasterPage.
+	// this hides the gap between the top bar and the page content.
+	if hp.CurrentPageID() == MainPageID {
```

Comment:
how do you know current app is mobile view ?

Reply:
I don't see any check mobile view for it

Reply:
my bad the mobile view comment shouldn't be there